packages/generated/array-buffer-byte-length/index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const is: (ab: unknown) => any;
```

Comment:
Don't know what happened here, but the returned value should be `boolean` of sort.

I guess the `uncurryThis` types is a bit too weird?

Reply:
<img width="567" alt="image" src="https://github.com/user-attachments/assets/7f8abdec-5ae1-4685-bec3-ed0ad0d0744b" />

The source itself is wrong typing.